widgetastic_manageiq/__init__.py

```diff
@@ -1038,6 +1038,28 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SSUIlist(Widget, ClickableMixin):
```

Comment:
1) Is it in patternfly? (then it belongs to wt.patternfly)
2) Is it just SSUI specific? (then it belongs into separate module)